### PR TITLE
fix(3000): last updated at overflowing on mobile dashboards

### PR DESCRIPTION
**Pull request by RhysSullivan**
## Problem

The "last updated at..." for dashboards on mobile overflows causing the screen to be able to be dragged horizontally

## Changes

Old:
image

New:
image

useResizeObserver.ts:
- Updates the generic to be `<T extends string>` - this is unrelated to this PR but it gives it the correct typing now
- Unrelated to this PR so I can remove it but I found it while developing and figured I'd fix it 

Old:
image

New:
image

Dashboard.scsss
- Created to allow for using breakpoints to change the flex direction depending on screen size

Dashboard.tsx
- space-x-4 was removed in the conversion to .scss styling as it wasn't actually applying any styling
- moves the tailwind styling to .scss

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

Manually checked in the local development environment

### CODE REVIEW

**Comment by RhysSullivan** [comment]
FYI I had to change 

`image: ghcr.io/posthog/kafka-container:v2.8.2` to `image: bitnami/kafka:3.5.1` in order to get docker compose to run - otherwise I ran into an authentication issue with accessing the container. Didn't include that change in this PR though as I was unsure if it was an error on my end

File: frontend/src/scenes/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -110,7 +112,7 @@ function DashboardScene(): JSX.Element {
                 <EmptyDashboardComponent loading={itemsLoading} canEdit={canEditDashboard} />
             ) : (
                 <div>
-                    <div className="flex space-x-4 justify-between">
+                    <div className="dashboard-controls">
```
Comment (Twixes) [inline frontend/src/scenes/dashboard/Dashboard.tsx:115]: We want to stick to utility classes these days as opposed to SCSS files. In fact, if you rebase on top of current `master`, you'll be able to use full Tailwind here. 😄 Including breakpoints, e.g. `md:flex-row` – though in this case perhaps flex wrapping would do the job?
```suggestion
                    <div className="flex gap-2 items-center justify-between flex-wrap">
```

**Review by Twixes — CHANGES_REQUESTED** [review]
Thanks for the contribution @RhysSullivan! Just one request from me is that we keep using utility classes here

**Review by Twixes — APPROVED** [review]
Oh, totally missed this last week, sorry about the delay.
Looks good, let's get this in!

**Pull request merged**